lib/streamlit/elements/media.py

```diff
@@ -52,13 +55,22 @@ def audio(
         data : str, bytes, BytesIO, numpy.ndarray, or file opened with
                 io.open().
             Raw audio data, filename, or a URL pointing to the file to load.
-            Numpy arrays and raw data formats must include all necessary file
+            Raw data formats must include all necessary file
```

Comment:
Raw data formats must include all necessary file headers to match the file format specified via `format`.